OMCompiler/Compiler/BackEnd/RemoveSimpleEquations.mo

```diff
@@ -192,6 +194,78 @@ algorithm
   end if;
 end removeVerySimpleEquations;
 
+public function fixAliasVarsVariablity
+" This is a workaround for #5673
+  This module traverses all alias variables and double-checks if the referenced alias variables variablitly
+  is parameter or not and changes variablilty of alias variable to BackendDAE.PARAM()."
+  input BackendDAE.BackendDAE inDAE;
+  output BackendDAE.BackendDAE outDAE;
+protected
+  BackendDAE.Variables aliasVars, systvars, globalKnownVars;
+  DAE.Exp binding;
+  DAE.ComponentRef cr;
+  Boolean paramOrConst, const;
+  BackendDAE.Shared shared;
+  list<BackendDAE.Var> referencevar, knownVarList={}, aliasVarList={};
+  BackendDAE.Var tempvar;
+algorithm
+  aliasVars := BackendDAEUtil.getAliasVars(inDAE);
+  systvars := BackendVariable.listVar(BackendVariable.equationSystemsVarsLst(inDAE.eqs));
+
+  //BackendDump.dumpVariables(aliasVars, "aliasVariable-Actual");
+  for var in BackendVariable.varList(aliasVars) loop
+    binding := BackendVariable.varBindExp(var);
+    {cr} := Expression.getAllCrefs(binding);
```

Comment:
This should probably be a list of crefs, even though it is the binding of an alias.